### ISSUE TITLE
The most recent version does not work at all

### ISSUE THREAD

**Issue by PyhaTanja** [ybrandvain/polyembryony#3]

**Comment by PyhaTanja** [comment]
Error in runSim(n.gen = 10, fitness.effects = 1, dom.effects = 0, gen.after.loss = 15,  : 
  object 'gen.summary' not found

**Comment by ybrandvain** [comment]
Shut sorry will try to fix

**Comment by ybrandvain** [comment]
Ok add line
gen.summary <- list()
On line 215
I’m about to take off and have no internet so can’t push till tonight but
should be and easy fix
